app/controllers/projects_controller.rb

```diff
@@ -343,6 +343,6 @@ def return_context
     render_404 && return if @project.blank?
     return unless @project.deleted?
     flash[:alert] = 'The project you requested had been deleted.'
-    redirect_to user_path(@user)
+    check_redirect "/#{@user.username}"
```

Comment:
I did NOT do those changes... They're just ready to be done and don't worry... I'm pretty sure they solve it :)